ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java

```diff
@@ -77,4 +80,15 @@ private static void makeAccessible(final Field field) {
 		}
 	}
 
+	public static List<Field> getDeclaredFieldsUpToParent(Class<?> currentClass, Class<?> exclusiveParent) {
+		List<Field> currentClassFields = newArrayList(currentClass.getDeclaredFields());
+		Class<?> parentClass = currentClass.getSuperclass();
+
+		if (parentClass != null && !(parentClass.equals(exclusiveParent))) {
+			List<Field> parentClassFields = getDeclaredFieldsUpToParent(parentClass, exclusiveParent);
```

Comment:
Is should be recursive call?

Reply:
If exist function that get all parent class, i can use loop instead of recursive.
i'll check this.

Reply:
In java, a class allow only one parent class so we don't need recursive call.. loop is enough.